Northstar.CustomServers/mod/scripts/vscripts/_utility.gnut

```diff
@@ -4012,38 +4012,33 @@ int function GameTime_TimeLeftMinutes()
 	if ( GetGameState() == eGameState.Prematch )
 		return int( ( expect float( GetServerVar( "gameStartTime" ) ) - Time()) / 60.0 )
 
-	return floor( GameTime_TimeLimitMinutes() - GameTime_PlayingTime() / 60 ).tointeger()
+	return floor( GameTime_PlayingTime() / 60 ).tointeger()
```

Comment:
is this correct? why not check the time limit if you're trying to get the time left?

Reply:
Getting the time limit would render this function useless if in-game time was ever changed manually via scripts